frontend/src/components/core/Input.tsx

```diff
@@ -1,9 +1,14 @@
 import styled from 'styled-components';
 
-const Input = styled.input`
+type Dimensions = {
+  width?: string,
+  height?: string,
+};
+
+const Input = styled.input<Dimensions>`
```

Comment:
I think all implementations of `Input` should implement the `font-family: ${({ theme }) => theme.font}` (the last one is currently not, but I think it should be added!). Could you add that line to this styled component, and remove it from the below ones?

Reply:
Sounds good, added - I left it in though for PureTextInputTitle since that one doesn't inherit Input (and changing it to inherit might cause some of the styles to clash)